.github/workflows/ci_compatible.yml

```
@@ -56,7 +58,7 @@ jobs:
       max-parallel: 25
       fail-fast: false
       matrix: ${{ fromJSON(needs.generator.outputs.matrix) }}
-    timeout-minutes: 35
+    timeout-minutes: 180
```

(possible reminder) Is this update expected? It seems like a significant change 👀

---

```suggestion
    timeout-minutes: 120
```
well, 3h is large but running TM full testing take on Mac tas over an hour and sometimes even 1.5h